cost/azure/databricks/rightsize_compute/azure_databricks_rightsize_compute.pt

```diff
@@ -1026,7 +1054,7 @@ script "js_idle_and_underutil_instances", type: "javascript" do
 
   applied_policy_name = ds_applied_policy["name"]
 
-  _.each(ds_filtered_workspace_virtualmachines_with_costs, function(vm) {
+  getIterator(ds_filtered_workspace_virtualmachines_with_costs).Each(function(vm, idx) {
     // Set the Applied Policy Name which is used in the summary_template/detail_template output
     vm["policy_name"] = applied_policy_name
     // Get the utilization metrics that we are going to use for comparison
```

Comment:
Line 1193 and Line 1354 -

Not caused by this change but see Errors - 
> 
"Recommendation policy has export that is missing required fields. These fields are scraped by the Flexera platform for dashboards:

Line 1193: accountID, accountName, resourceID, service, savingsCurrency
Line 1354: accountID, accountName, resourceID, service, savingsCurrency" 
>

e.g., line 1201 should be `accountID` instead of `subscriptionId` (and same for `subscriptionName`); `resourceID`, `service` and `savingsCurrency` fields need to be added